### DISCUSSION TITLE
Lodestar Standup 2024-31 (August 13, 2024)

### DISCUSSION THREAD

**Discussion by philknows** [ChainSafe/lodestar#7008] [Standups]
### Meeting Info
[August 13, 2024, 14:00-15:00 UTC](https://savvytime.com/converter/utc-to-germany-berlin-united-kingdom-london-ny-new-york-city-ca-san-francisco-china-shanghai-japan-tokyo-australia-sydney/aug-13-2024/2pm)
Zoom: shared on #lodestar-general ChainSafe Discord channel at 14:00 UTC

### Agenda
- Electra merge update
- Ansible playbooks/roles migration
  - What should be migrated
  - What should be refactored (and why)
  - Who should do it
- js-libp2p Wishlist items
- Open Discussion
- Team Updates
- Closing Remarks

### Instructions
Please post here any topics to be added for Standup discussions. Examples of agenda items are issues, discussions, proposals which require feedback, consensus from the team and/or community members.

**Comment by philknows** [comment]
Adding https://github.com/ChainSafe/lodestar/issues/7011 to discuss if there are additional ideas to note.

**Comment by wemeetagain** [comment]
Discussion about the possible js-libp2p retreat in September. If anyone has any wishlist items or things to prioritize on the roadmap, please share.

**Reply by wemeetagain** [reply]
- quic in the browser? future of quic in js-libp2p
- observability, any tooling to promote that, considerations for how to determine peer quality

**Comment by wemeetagain** [comment]
Discussion about managing our ansible playbooks / roles

**Comment by philknows** [comment]
Closing for #7040 7040
